apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -594,7 +605,8 @@ export class ChangeAction extends BaseRushAction {
    * Writes change files to the common/changes folder. Will prompt for overwrite if file already exists.
    */
   private async _writeChangeFiles(
-    promptModule: inquirer.PromptModule,
+    // eslint-disable-next-line @typescript-eslint/typedef
```

Comment:
As I explained with `IYarnLockfileParseResult`, just use a regular `import` with a TODO comment saying to convert to  "import type" later.  *Don't remove our type annotations.*